src/main/java/net/fabricmc/loader/entrypoint/minecraft/hooks/EntrypointUtils.java

```diff
@@ -38,26 +40,24 @@
 	private static <T> void invoke0(String name, Class<T> type, Consumer<? super T> invoker) {
 		@SuppressWarnings("deprecation")
 		FabricLoader loader = FabricLoader.INSTANCE;
```

Comment:
Wouldn't it be as good a time as any to replace that reference with the API instance ?

Reply:
This is an internal call within invoke0 so it isn't imparative. But I easily could with no real issues.

Reply:
I mean, SuppressWarnings is never pretty. On the other hand, it has basically nothing to do with the base PR so whatever.

Reply:
I'll fix that tomorrow since it literally costs us nothing to change that.

Reply:
This one is better than a cast; should be left as-is.

Reply:
Did we want this resolved before merge or is this specific issue unneeded.